examples/conda_build_config.yaml

```diff
@@ -0,0 +1,38 @@
+# this should always be similar to conda-forge's pinnings:
+# https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml
+
+pin_run_as_build:
+  boost:
+    max_pin: x.x.x
+  boost-cpp:
+    max_pin: x.x.x
+
+# blas
+libblas:
```

Comment:
wow, I like this way to lock down the version.

Reply:
Full credits to @wolfv!